packages/framework-provider-aws/src/constants.ts

```diff
@@ -22,4 +22,6 @@ export const connectionsStoreAttributes = {
 export const environmentVarNames = {
   restAPIURL: 'BOOSTER_REST_API_URL',
   websocketAPIURL: 'BOOSTER_WEBSOCKET_API_URL',
+  jwtIssuer: JWT_ENV_VARS.BOOSTER_JWT_ISSUER,
+  jwksUri: JWT_ENV_VARS.BOOSTER_JWKS_URI,
```

Comment:
I'd say you don't need `jwtIssuer` nor `jwkUri` constants, as you already have those constant in the generic types package (`JWT_ENV_VARS.*`)

You can import `'@boostercloud/framework-types'` from "aws" and "aws-infrastructure" perfectly 😃

Reply:
Umm, I'm doing that at the beginning of the file and the `JWT_ENV_VARS` are defined at `framework-types` level like that:

```
export const JWT_ENV_VARS = {
  BOOSTER_JWT_ISSUER: 'BOOSTER_JWT_ISSUER',
  BOOSTER_JWKS_URI: 'BOOSTER_JWKS_URI',
}
```

What I'm missing? @alvaroloes

Reply:
Ohh I see, forget my last comment